ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/DeploymentsAtMostRiskTable.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+
+import { ListDeployment } from 'types/deployment.proto';
+import { networkBasePath, riskBasePath } from 'routePaths';
+import { SearchFilter } from 'types/search';
+import { getUrlQueryStringForSearchFilter } from 'utils/searchUtils';
+
+const columnNames = {
+    deployment: 'Deployment',
+    resourceLocation: 'Resource location',
+    riskPriority: 'Risk priority',
+};
+
+function riskPageLinkToDeployment(id: string, name: string, searchFilter: SearchFilter): string {
+    const query = getUrlQueryStringForSearchFilter({
+        ...searchFilter,
+        Deployment: name,
```

Comment:
Aha, `Deployment: name` is necessary only because Risk has classic table/panel instead of risk deployment page for /main/risk/id path?

Reply:
Yeah, the goal here was to filter the list down to reduce confusion for the user since there isn't a detail page for a single risk item, as well as give them the option to clear "Deployment" from their search filter if needed.